lib/jekyll/page.rb

```diff
@@ -18,37 +9,36 @@ class Page
       name
       path
       url
-    )
+    ).freeze
```

Comment:
These attributes should be specified in the `Drop`. Let's use `DocumentDrop` if possible.